libr/io/p/io_r2k.c

```diff
@@ -65,7 +65,7 @@ static int r2k__close(RIODesc *fd) {
 	}
 #elif __linux__
 	if (fd) {
-		close (fd->fd);
+		close (fd->user);
```

Comment:
that is for RIO right? And I am using RIODesc but it does not have *user field as condret suggested in the issue. Might be either he is updating the RIODesc or he mistakenly wrote it in wrong way.